### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: python/test/t_ContinuousPC_std.py [hunk 1]
```diff
@@ -29,8 +29,8 @@ def testSpecificInstance():
     # print(skel.toDot())
 
     dag = learner.learnDAG()
-    print(dag.toDot(), flush=True)
-
+    print(dag.toDot())
+    sys.stdout.flush()
```
Comment (phwuil) [inline python/test/t_ContinuousPC_std.py]:
```suggestion
print(dag.toDot(),flush=True)
```

Comment (jschueller) [reply python/test/t_ContinuousPC_std.py]: I tried it, unfortunately that wouldnt work on py2k

Comment (phwuil) [reply python/test/t_ContinuousPC_std.py]: oh ... 
```python
from __future__ import print_function
``` 
??
(in order to have py3 (rather than py2)'s code)

Comment (jschueller) [reply python/test/t_ContinuousPC_std.py]: the flush argument is not available, it was introduced much later, in python 3.4

**Pull request merged**